qemu/tests/pvpanic_memory_leak.py

```diff
@@ -0,0 +1,35 @@
+from virttest import error_context
+from virttest import utils_test
+from provider import win_driver_utils
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Virtio-win-pvpanic memory leak checking test:
+    1) Start a windows guest with device: Pvpanic
+    2) Verifying the driver via 'verifier' firstly, then reboot
+    3) Re-enter the system, disable the pvpanic and reboot again
+    4) checking memory leak after few seconds: BSOD or vm dead
+       if leaking occurs
+    5) Enabled pvpanic and disabled driver verifier
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    timeout = float(params.get("login_timeout", 240))
```

Comment:
This is unnecessary, as below wait_for_login() will check verify_alive() anyway. And I don't see why we need an specific timeout defined here besides its default value.